### PR TITLE
Env var for restart timeout

### PR DESCRIPTION
**Pull request by rudle**
An internal request for exposing the restart_timeout has prompted this change. The alternative is having the customer parse the manifest (bad) & copy the knowledge of the default timeout (very bad!)

### CODE REVIEW

File: pkg/pods/pod.go [hunk 1]
```diff
@@ -42,13 +42,14 @@ var (
...
-	PodUniqueKeyEnvVar       = "POD_UNIQUE_KEY"
-	PlatformConfigPathEnvVar = "PLATFORM_CONFIG_PATH"
+	ConfigPathEnvVar               = "CONFIG_PATH"
+	LaunchableIDEnvVar             = "LAUNCHABLE_ID"
+	LaunchableRootEnvVar           = "LAUNCHABLE_ROOT"
+	PodIDEnvVar                    = "POD_ID"
+	PodHomeEnvVar                  = "POD_HOME"
+	PodUniqueKeyEnvVar             = "POD_UNIQUE_KEY"
+	PlatformConfigPathEnvVar       = "PLATFORM_CONFIG_PATH"
+	LaunchableRestartTimeoutEnvVar = "RESTART_TIMOUT"
```
Comment (mpuncel) [inline pkg/pods/pod.go]: s/TIMOUT/TIMEOUT

File: pkg/pods/pod.go [hunk 2]
```diff
@@ -602,7 +603,11 @@ func (pod *Pod) setupConfig(manifest manifest.Manifest, launchables []launch.Lau
 		if err != nil {
 			return err
 		}
-		err = writeEnvFile(launchable.EnvDir(), "LAUNCHABLE_ROOT", launchable.InstallDir(), uid, gid)
+		err = writeEnvFile(launchable.EnvDir(), LaunchableRootEnvVar, launchable.InstallDir(), uid, gid)
+		if err != nil {
+			return err
+		}
+		err = writeEnvFile(launchable.EnvDir(), LaunchableRestartTimeoutEnvVar, fmt.Sprintf("%s", launchable.GetRestartTimeout()*time.Second), uid, gid)
```
Comment (mpuncel) [inline pkg/pods/pod.go]: have you checked how this turns into a string? seems like it's already a `time.Duration` so you probably don't need to `*time.Second`

Comment (rudle) [reply pkg/pods/pod.go]: fair cop. Here's what go thinks: https://play.golang.org/p/wJQzqbhN-T

I like the explicit int64() conversion - WDYT?

**Comment by rudle** [comment]
PTAL

**Pull request merged**